src/org/omegat/gui/dictionaries/DictionaryPaneExtension.java

```diff
@@ -0,0 +1,50 @@
+/**************************************************************************
+ OmegaT - Computer Assisted Translation (CAT) tool
+          with fuzzy matching, translation memory, keyword search,
+          glossaries, and translation leveraging into updated projects.
+
+ Copyright (C) 2021 Hiroshi Miura
+               Home page: http://www.omegat.org/
+               Support center: https://omegat.org/support
+
+ This file is part of OmegaT.
+
+ OmegaT is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ OmegaT is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ **************************************************************************/
+package org.omegat.gui.dictionaries;
+
+import tokyo.northside.protocol.URLProtocolHandler;
+
+/**
+ * @author Hiroshi Miura
+ */
+public class DictionaryPaneExtension {
```

Comment:
Sorry for the bikeshedding, but this naming implies that it could do all sorts of things to the Dictionary pane. I would rather have separate extensions for separate concerns.

Could we call it something more focused? Like `DictionaryMediaSupport` or something?

Reply:
Actually, doesn't this affect more than just the Dictionary pane? It should affect all `JTextPane`s that have HTML documents, right? So we should probably change the package as well. What about `org.omegat.util.gui.HtmlMediaSupport`?

Reply:
> org.omegat.util.gui.HtmlMediaSupport

I'd like to take it.